backend/src/Agent/SlackNotificationAgent.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace App\Agent;
+
+use App\DTO\AgentMessage;
+use App\Entity\ProjectAgentConnection;
+use App\Entity\Task;
+use App\Entity\TaskResult;
+use App\Enum\AgentFieldType;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class SlackNotificationAgent extends AbstractAgent
+{
+    public function __construct(
+        private readonly HttpClientInterface $httpClient,
+    ) {
+    }
+
+    /**
+     * @inheritDoc
+     * @param AgentMessage[] $messages
+     */
+    public function processTask(
+        Task $task,
+        ProjectAgentConnection $connection,
+        array &$messages,
+    ): ?TaskResult {
+        $msg = $this->replaceMessageVariables(
+            $connection->config['messageText']
```

Comment:
Double-check the default message formatting here. The default message mentions `{taskStatus}` which isn't being replaced in `replaceMessageVariables` function, leading to a placeholder in the output message.